modules/infra/cli/loopback.c

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+
+#include <gr_api.h>
+#include <gr_cli.h>
+#include <gr_cli_iface.h>
+#include <gr_infra.h>
+
+static void loop_show(const struct gr_api_client *, const struct gr_iface *) { }
+
+static void
+loop_list_info(const struct gr_api_client *, const struct gr_iface *, char *buf, size_t len) {
+	snprintf(buf, len, "%s", "");
```

Comment:
Probably not needed.

Reply:
buf is already filled with the parent function, so it is required to clear it.

Reply:
Oh I get it, `buf` should be reset before calling `type->list_info()` here:

https://github.com/DPDK/grout/blob/bd52b2c275e23812e18d39601e179256551ab19f/modules/infra/cli/iface.c#L298